cmd/wallet-web/src/translations/CHAPI/fr.json

```diff
@@ -14,5 +14,12 @@
       "body": "Something went wrong. Please, try again.",
       "tryAgain": "Try Again"
     }
+  },
+  "Issue": {
+    "Error": {
+      "heading": "Error",
+      "body": "Something went wrong. Please, try again.",
+      "tryAgain": "Try Again"
```

Comment:
Can we add a TODO/file an issue to update these with French values?

Reply:
part of #1336 as mentioned in PR